### DISCUSSION TITLE
understanding the stats

### DISCUSSION THREAD

**Discussion by tcurdt** [transmission/transmission#5985] [Q&A]
When I am running transmission and the stats say:

- seed: 0/3
- peers: 0/4

IIUC this means, 3 people have the full copy and are seeding - but none have allowed a connection.
With the peers it also means 0 of the 4 are connected - but it might just mean there is nothing more to exchange.

Is that correct?

And as long as the incoming connection port is successful, there is nothing else one can do to convince a seeder to allow the connection.

Correct?

**Comment by tearfur** [comment]
`x/y` it means we are actively transferring data with `x` peers out of `y` peers we are currently connected to, no matter when seeding or leeching. At least this is how it is in the WebUI, and I'm pretty sure all clients are supposed to be consistent.

**Reply by tcurdt** [reply]
Thanks - but the question had a different angle:
When it says 0/3 seeds, it means that there are 3 people out there having a full copy - but 0 are offering me data.
Correct?

**Reply by tearfur** [reply]
Yes.

**Reply by tcurdt** [reply]
And even with an open port there is nothing else I can do to convince the seeder to hand me out the missing data.
Correct?

**Reply by tearfur** [reply]
AFAICT yes.

If we get into a bit more technical details, what's probably happening is you have no pieces to offer, and by the tit-for-tat bartering algorithm, you can only wait for someone to optimistically unchoke you.

The wait time can be less if Tranmission handles allowed fast messages defined in BEP-6, but that will take a PR from someone.
